scripts/build_release.sh

```
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+projectRootPath=$(realpath "$0" | sed 's|\(.*\)/.*|\1|' | cd ../ | pwd)
```

Shellcheck (0.10.0)

[warning] Piping to 'cd', a command that doesn't read stdin. Wrong command or missing xargs?

(SC2216)